app/mailers/subscription_mailer.rb

```diff
@@ -41,7 +41,7 @@ def notify_tag_added(node, tag, current_user)
     final_users_to_email.each do |user|
       @user = user
       unless user.id == current_user.id 
-         mail(to: user.email, subject: "Added: #{node.title}").deliver
+         mail(to: user.email, subject: "New tag added on #{node.title}")
```

Comment:
Ah, we'd meant to change this but it looks reverted?

Reply:
Sure I will change it. Thanks!!